setup.py

```diff
@@ -31,12 +31,20 @@
     "django": (
         [
             "django-cors-headers==3.11.0",
-            "django",
+            "django==3.2.12",
```

Comment:
This should not have a version - we want to always tests the integration test with the latest version of django 3, so that we know if things have broken

Reply:
done.